util/schema/registry_schema.json

```diff
@@ -96,9 +96,33 @@
       "suggest": false
     },
     "domain": {
-      "level": "warning",
-      "description": "'domain' is a recommended field. The value is the scientific topic area addressed by the ontology.",
-      "type": "string"
+      "level": "error",
+      "description": "'domain' is a mandatory field. The value is the scientific topic area addressed by the ontology.",
+      "type": "string",
+      "enum": [
+        "upper",
+        "anatomy and development",
+        "health",
+        "phenotype",
+        "experiments",
+        "environment",
+        "biochemistry",
+        "microbiology",
+        "agriculture",
+        "diet, metabolomics and nutrition",
+        "biological systems",
+        "information technology",
+        "organisms",
+        "information",
+        "other"
```

Comment:
the only "other" is the digital organisms one -> would be better to just reclassify as "organisms" then remove "other" so things don't get added without thinking this through very carefully

Reply:
Replaced by new category "simulation" as @alanruttenberg suggested.